lib/event_tracer/helpers/prometheus_helper.rb

```diff
@@ -0,0 +1,15 @@
+module EventTracer
+  module Helpers
+    module PrometheusHelper
+      module_function
+
+      def register_prometheus_metrics(prometheus_registry, name, labels)
+        prometheus_registry.counter(
```

Comment:
Should we allow overwriting `counter` here?

Reply:
Oh yes true. Note that `gauge` is a bit more complicated to register:
```
free_disk_space = registry.gauge(:free_disk_space_bytes,
                                docstring: "Free disk space, in bytes",
                                store_settings: { aggregation: :max })
```

so you'll have to merge in the `store_settings` (I think we can just add a constant for the value for simplicity)